socorro/unittest/conftest.py

```diff
@@ -25,3 +28,165 @@ def test_something(metricsmock):
 
     """
     return MetricsMock()
+
+
+class DjangoTablesMixin(object):
```

Comment:
This centralizes maintenance of Django-managed tables for non-Django tests. This is a temporary thing as most of the tests that need this are crontabber jobs and those will eventually either go away or get rewritten as Django things. The only one that should stick around is the BetaVersionRule tests. We can figure out what to do with those then.